suppress_useless_warnings.props

```diff
@@ -5,7 +5,7 @@
   <PropertyGroup />
   <ItemDefinitionGroup>
     <ClCompile>
-      <DisableSpecificWarnings>4018;4373;4503;4554;4566;4996;%(DisableSpecificWarnings)</DisableSpecificWarnings>
+      <DisableSpecificWarnings>4018;4244;4373;4503;4554;4566;4996;%(DisableSpecificWarnings)</DisableSpecificWarnings>
```

Comment:
Do we really need this?  This warning is useful because these data losses are extremely hard to debug.  I'd rather have static_cast<> in the code (that's what I did for de/serialization).